app/assets/stylesheets/main.scss

```diff
@@ -647,4 +647,25 @@ a {
 .email-input {
   padding-inline-start: 1.375rem !important;
   padding-inline-end:   1.375rem !important;
+}
+
+// Menu modal
+
+.backdrop-logout-modal{
+  background: #A6A1A1 !important;
+  filter: blur(50px) !important;
```

Comment:
In general it's recommended not to use `!important` very often in CSS, so please only use them if absolutely necessary.